src/components/Card/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { Card, CardProps, CardRatioOptions } from './Card';
+export { OverlappingCard } from './OverlappingCard';
```

Comment:
@shlomitc i want to simplify the import of *.st.css files from another .st.css files
is exporting the style object from here is ok
so devs could just use `:import {
        -st-from: "wix-ui-tpa/Card";
        -st-named: CardStyle;
    }`
instead of `:import {
  -st-from: "wix-ui-tpa/src/components/Button/Button.st.css";
  -st-default: TPAButton;
}`

Reply:
I'll make it possible in a different PR, but if there is more than one exported component - it should have a separate folder entry

Reply:
https://github.com/wix/wix-ui-tpa/pull/56